test/tools/jsontests/StateTests.cpp

```diff
@@ -49,7 +49,7 @@ void doStateTests(json_spirit::mValue& _v, bool _fillin)
 		string testname = i.first;
 		json_spirit::mObject& o = i.second.get_obj();
 
-		if (_fillin)
+		if (_fillin && !Options::get().createRandomTest)
```

Comment:
the design of the tests was originally to run a function doTests  and  if passed a flag  filling it will generate the test, otherwise it will run a test. because the code for testing and generating a test is pretty much the same.